t/unit/conftest.py

```diff
@@ -18,7 +18,7 @@
 # in case user did not do the `python setup.py develop` yet,
 # that installs the pytest plugin into the setuptools registry.
 from celery.contrib.pytest import (
-    celery_app, celery_enable_logging, depends_on_current_app,
+    celery_app, celery_enable_logging, depends_on_current_app, celery_parameters
```

Comment:
Needed for testing to work, current version of OG celery also imports `celery_parameters`.